whats_new/ose_3_0_release_notes.adoc

```diff
@@ -93,3 +93,149 @@ link:../rest_api/kubernetes_v1.html#v1-rbdvolumesource[RADOS (Ceph)].
 - https://bugzilla.redhat.com/show_bug.cgi?id=1233540[BZ#1233540]: Persistent
 Volume does not get recycled when the persistentVolumeReclaimPolicy is
 "Recycle"
+
+These are the release notes for xPaaS images on OpenShift Enterprise
+3.0. For the release notes for OpenShift Enterprise 3.0, please refer
+https://access.redhat.com/beta/documentation/en/openshift-enterprise-30-whats-new#release-notes[here].
+
+xPaaS images for OpenShift Enterprise 3.0 are provided for Red Hat JBoss
+Enterprise Application Platform, Red Hat JBoss Web Server and Red Hat
+JBoss A-MQ.
+
+== xPaaS Images
+
+=== xPaaS Image for Red Hat JBoss EAP
+
+
+With this release, Red Hat JBoss EAP is available as a containerized
+xPaaS image that is designed for use with OpenShift Enterprise 3.0.
+
+However, there are significant differences in supported configurations
+and functionality in the JBoss EAP xPaaS image compared to the regular
+release of JBoss EAP. To learn about these differences, please read
+https://access.redhat.com/beta/documentation/en/openshift-enterprise-30-using-images#jboss-eap[this documentation].
```

Comment:
Only referring to the CP links to OSE docs, though, not the related non-OpenShift-specific middleware docs.